appium-dotnet-driver/Appium/AppiumCommandExecutionHelper.cs

```diff
@@ -58,6 +65,77 @@ public static void HideKeyboard(IExecuteMethod executeMethod, string strategy =
         public static void Lock(IExecuteMethod executeMethod, int seconds) =>
             executeMethod.Execute(AppiumDriverCommand.LockDevice,
                 new Dictionary<string, object>()
-                    {["seconds"] = seconds});
+                { ["seconds"] = seconds });
+
+        public static string GetClipboard(IExecuteMethod executeMethod, ClipboardContentType clipboardContentType)
+        {
+            return (string)executeMethod.Execute(AppiumDriverCommand.GetClipboard,
+                PrepareArgument("contentType", clipboardContentType.ToString().ToLower())).Value;
+        }
+
+        public static string GetClipboardText(IExecuteMethod executeMethod)
+        {
+            var encodedContentBytes = Convert.FromBase64String(GetClipboard(executeMethod, ClipboardContentType.PlainText));
+            return Encoding.UTF8.GetString(encodedContentBytes);
+        }
+
+        public static string SetClipboard(IExecuteMethod executeMethod, ClipboardContentType clipboardContentType, string base64Content)
```

Comment:
I find the design of this method is not very obvious since `base64Content` can be an actual base64-encoded content or a normal string

Reply:
I'd also expect urls to be represented by special type instead of raw strings

Reply:
I believe this needs to be a string in order to support .NetStandard, as there is no URL type.